index.js

```diff
@@ -0,0 +1,33 @@
+var geek = [];
```

Comment:
Use a better word for this variable, think what domain you are working on and name the variables accordinly